cmake/deps.txt

```diff
@@ -27,7 +27,7 @@ flatbuffers;https://github.com/google/flatbuffers/archive/refs/tags/v23.5.26.zip
 fp16;https://github.com/Maratyszcza/FP16/archive/0a92994d729ff76a58f692d3028ca1b64b145d91.zip;b985f6985a05a1c03ff1bb71190f66d8f98a1494
 fxdiv;https://github.com/Maratyszcza/FXdiv/archive/63058eff77e11aa15bf531df5dd34395ec3017c8.zip;a5658f4036402dbca7cebee32be57fb8149811e1
 google_benchmark;https://github.com/google/benchmark/archive/refs/tags/v1.8.5.zip;cd47d3d272faf353600c8cc2fdec2b52d6f69177
-google_nsync;https://github.com/google/nsync/archive/refs/tags/1.26.0.zip;5e7c00ef6bf5b787386fc040067903ec774e2752
+google_nsync;https://github.com/google/nsync/archive/refs/tags/1.29.0.zip;acc46ec0bc736da1cf581c9bb2700f9cea210b6c
```

Comment:
Sorry you cannot make changes to this file. It has to be changed by someone who has write access to this repo. 
If you could wait for a few days, I will create a separated PR for handling this change.

Reply:
Understood and that would be great, thank you.

In the next couple of days, I'll resolve the CI build fail issues from earlier. Then I can rebase against your deps.txt changes when they're in and let you know.

Reply:
Resolved by #20413